lib/api/v3/work_packages/work_packages_api.rb

```diff
@@ -98,8 +108,11 @@ def write_request_valid?
                                                             send_notifications)
 
               if write_request_valid? && update_service.save
-                @representer.represented.reload
-                @representer
+                work_package = @representer.represented
+                work_package.reload
+                @representer = WorkPackages::WorkPackageRepresenter.create(
+                  work_package,
+                  current_user: current_user)
```

Comment:
@NobodysNightmare can't you simply take the work_package out of the `@work_package` variable? Especially if you are reloading it anyway.